example/src/App-ParentRef.tsx

```diff
@@ -0,0 +1,103 @@
+import { CanvasForm, CanvasSpace } from 'pts'
+import React, { useRef, createRef } from 'react'
+import { PtsCanvas, HandleAnimateFn, HandleActionFn } from 'react-pts-canvas'
+import './App.css'
+
+/**
+ * Brief example of passing refs down to pointsCanvas for access in parent component
+ */
+const App: React.FC = () => {
+  // Use a useRef instead of useState hook here as the canvas
+  // Doesn't re-render based on the radius, it only affects the handleAnimate
+  const radiusRef = useRef(50)
+  const canvRef = createRef<HTMLCanvasElement>()
+  const spaceRef = useRef<CanvasSpace>()
```

Comment:
The `spaceRef` and `formRef` really confuses me since I'm not yet familiar with the React patterns. Maybe we can add more comments here to help user understand how this works?

Reply:
I'm wondering if these props are really needed. If the parent component needs to access the space or the form, they can get it from `onStart`, isn't it?

Reply:
@tibotiber that is true, I suppose the difference is whether or not they want access to the actual ref being used within the component? I guess there aren't to many use cases for that, so happy to remove

Reply:
Again, seems like a rare use case and just makes things more confusing. Removed for now.